test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2335,6 +2352,7 @@ public void testMetadataColumns_clobNull() throws Exception {
 
   @Test
   public void testMetadataColumns_blob() throws Exception {
+    assumeFalse("SQL Server does not support blobs", is(SQLSERVER));
```

Comment:
We should be able to delete this check and the one in blobNull, below, shouldn't we?